go-controller/pkg/ovn/secondary_layer3_network_controller.go

```diff
@@ -420,8 +420,12 @@ func (oc *SecondaryLayer3NetworkController) newRetryFramework(
 // Start starts the secondary layer3 controller, handles all events and creates all needed logical entities
 func (oc *SecondaryLayer3NetworkController) Start(ctx context.Context) error {
 	klog.Infof("Start secondary %s network controller of network %s", oc.TopologyType(), oc.GetNetworkName())
-
-	if err := oc.Init(ctx); err != nil {
+	networkID, err := oc.getNetworkID()
+	if err != nil {
+		return fmt.Errorf("unable to set networkID on secondary L3 controller for network %s", oc.GetNetworkName())
```

Comment:
can you also include error to the message.

Reply:
done

Reply:
nice catch